README.md

```diff
@@ -1 +1,56 @@
-# cr-mobile-menu
\ No newline at end of file
+# Mobile Menu
+
+> This project is for practicing code review.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Browser
+- A version control like git
+- A text editor
+
+### Setup
+
+- First clone or download using the link [https://github.com/binhussen/cr-mobile-menu](https://github.com/binhussen/cr-mobile-menu).
```

Comment:
- [**OPTIONAL**] Microverse team has provided a new version of the README file. Unfortunately you didn't use it. It could be better if you update your README file accordingly.